dlc_developer_config.toml

```diff
@@ -74,7 +74,7 @@ ec2_tests_on_heavy_instances = false
 sagemaker_local_tests = false
 
 # run standard sagemaker remote tests from test/sagemaker_tests
-sagemaker_remote_tests = false
+sagemaker_remote_tests = "standard"
```

Comment:
Also what does remote tests do? I do not see from the PR template or README.md to enable this?

Reply:
the example PR in out DLC vuln runbook talks about these tests. I'll share with you the reference offline